configuration/driver/icedrv_arrays_column.F90

```diff
@@ -235,15 +235,15 @@ module icedrv_arrays_column
          snow_bio_net   ! depth integrated snow tracer (mmol/m^2)
 
       ! floe size distribution
-      real(kind=dbl_kind), dimension(nfsd), public ::  &
+      real(kind=dbl_kind), dimension(nx,nfsd), public ::  &
          floe_rad_l,    &  ! fsd size lower bound in m (radius)
          floe_rad_c,    &  ! fsd size bin centre in m (radius)
          floe_binwidth     ! fsd size bin width in m (radius)
 
       real (kind=dbl_kind), dimension (nx), public :: &
          wave_sig_ht       ! significant height of waves (m)
 
-      real (kind=dbl_kind), dimension (nfreq), public :: &
+      real (kind=dbl_kind), dimension (nx,nfreq), public :: &
```

Comment:
They don't have to, but I thought it was a good idea.

Reply:
@erinethomas would this change be helpful for E3SM?  If it's not anticipated to be needed in the near future in E3SM or CESM (or other model -- everyone speak up please), I'd prefer to keep the arrays small.

Reply:
pinging @erinethomas on this question

Reply:
(sorry for delayed response) - the wave frequency space is fixed - so I do not foresee a need to have a grid dependency.... unless we want to go with spatially-varying spectral resolution in WW3 (It IS an idea that has come up .... but one I have no idea how to actually implement in ww3.... )

Reply:
if e3sm wishies to go this direction (spatially varying spectral resolution) -  it would take a massive re--write of ww3, and would be a DISTANT future goal (not something I plan on attempting in the near future)

Reply:
I am confused. We receive a three dimensional field from WW3 which is nx x ny x nfreq. This means we get a different 25 frequencies at every point on the globe. Am I missing something?